wherehows-common/src/main/java/wherehows/common/utils/JobsUtil.java

```diff
@@ -19,22 +19,27 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
 import java.util.Properties;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import org.apache.commons.configuration.Configuration;
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.commons.configuration.PropertiesConfiguration;
 import wherehows.common.Constant;
 
 
 public class JobsUtil {
 
   // Patterns for environmental variables resolution in jobs file.
-  private static final List<Pattern> ENV_VAR_PATTERNS = ImmutableList.<Pattern>builder()
-      .add(Pattern.compile("\\$(.+)")) // $ENV_VAR
-      .add(Pattern.compile("\\$\\{(.+)\\}")) // ${ENV_VAR}
-      .build();
+  private static final List<Pattern> ENV_VAR_PATTERNS =
```

Comment:
Should be able to drop this now?

Reply:
Yes removed it along with method and test